tests/js/integration/convert-tiff-images-in-expanded-bucket-to-jpeg-images.test.js

```diff
@@ -62,7 +62,7 @@ describe('Test convertTiffImagesInExpandedBucketToJpegImages', () => {
     process.env.DESTINATION_BUCKET = destinationBucket;
     process.env.ARCHIVE_FILE_NAME = 'biorxiv-685172.meca';
 
-    s3Utils.createBucket({Bucket: destinationBucket});
+    await s3Utils.createBucket({Bucket: destinationBucket});
 
     await extractArchivedFilesToBucket();
     fu.deleteFile = jest.fn();
```

Comment:
Could extract this and the same on line 40 into a `beforeEach()`

Reply:
Tried, but doesn't work as `fu.deleteFile` needs to be stubbed from this point onwards rather than for the whole method.